github-orgs/manifests/validate_config.py

```diff
@@ -29,7 +29,7 @@ def check_config(seld, config):
 
     # TODO(jlewi): We should load this in via config map
     # Check that each admin is in a whitelist set of admins.
-    allowed_admins = ["caniszczyk", "chensun", "google-admin", "googlebot",
+    allowed_admins = ["caniszczyk", "chensun", "googlebot",
```

Comment:
Can we also add KSC members as admins?

Reply:
Sure, but let's do it in another PR if this is okay with you.

Reply:
SGTM